modules/eventadd.php

```diff
@@ -533,6 +535,19 @@
     $SMARTY->assign('customerlist', $LMS->GetAllCustomerNames());
 }
 
+if (!empty($_GET['description'])) {
+    $event['description'] = filter_var($_GET['description'], FILTER_SANITIZE_SPECIAL_CHARS);
+}
```

Comment:
Zmieniamy metodę sanityzacji z frontend side (template-y) na backend side - skrypty php?

Reply:
Nie do końca zmieniamy - sanityzacja na backendzie jest wykonywana tylko w momencie użycia parametru zaszytego w URL.

Teoretycznie mogłoby tego fragmentu nie być  - ale to nie byłoby za bezpieczne bo PHP z tymi danymi uruchomiłby się - stąd dodana sanityzacja na backendzie.